app/tasks.py

```diff
@@ -12,6 +12,15 @@
 # Use 'celery -A reboot worker -l info'
 # Then in another terminal, run 'python manage.py runserver'
 # Make sure worker is running, then tassk will be queued by worker to parse the data.
+from celery import Celery, current_task
+import csv, datetime, zipfile
+from .models import Donor, Donation, Item
+from celery import shared_task
+import datetime, StringIO, os
```

Comment:
Same thing with StringIO?

Reply:
We should really clean up our requirements and imported packages